### PR TITLE
Send CallContext message to KEVM-VM

### PR DESCRIPTION
**Pull request by anvacaru**
-- Added Protobuf file to `lib/proto` and `google-protobuf` module to `package.json`
-- Removed most part of the logic from `runCall.js`  and replaced it with a tcp client which connects to the `kevm-vm` and sends a `hello` and a `CallContext` message

### CODE REVIEW

File: lib/runCall.js [hunk 1]
```diff
@@ -53,240 +55,93 @@ module.exports = function (opts, cb) {
...
-        })
-      } else {
-        stateManager.commit(cb)
-      }
-    }
+  function createEthereumConfig () {
+    var ethereumConfig = new msg_pb.EthereumConfig()
+    ethereumConfig.setMaxcodesize(new Uint8Array([96, 0]))
+    ethereumConfig.setAccountstartnonce(new Uint8Array([0]))
+    ethereumConfig.setFrontierblocknumber(new Uint8Array([0]))
```
Comment (dwightguth) [inline lib/runCall.js:126]: how does ethereumjs-vm choose which hard fork to use?

**Review by dwightguth — COMMENTED** [review]
looks okay to me, but we should discuss how we're going to deal with hard forks because that functionality does not seem to be implemented yet in this PR. We should take care of that now as the CallContext code isn't really done until it's dealt with.

File: lib/runCall.js (hunk 1 above)
Comment (anvacaru) [reply lib/runCall.js:126]: It looks like only three hardforks are currently supported: `byzantium`, `constantinople` and `petersburg`.  Ganache takes it as a command line argument at the startup https://github.com/ethereumjs/ethereumjs-vm/blob/b48b5c23524c41285f6e69667ca11b9e42133f21/lib/index.js#L41-L46

Comment (dwightguth) [reply lib/runCall.js:126]: In that case, let's leave this for now as it depends on updating the kevm plugin to support the new hardforks. We should put this on your list of tasks though.

**Review by dwightguth — APPROVED** [review]

**Comment by eviefp** [comment]
This also looks good to me. @anvacaru feel free to merge this PR whenever you feel it's ready.

**Pull request merged**